packages/given_when_then/test/example_widget_test_support.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:given_when_then/given_when_then.dart';
+
+Future<void> Function(WidgetTester) harness(
+    WidgetTestHarnessCallback<_ExampleWidgetTestHarness> callback) {
+  return (tester) =>
+      givenWhenThenWidgetTest(_ExampleWidgetTestHarness(tester), callback);
+}
+
+class _ExampleWidgetTestHarness extends WidgetTestHarness {
```

Comment:
we should consider adding an example where we apply a mixin to provide access to other G/W/T's

future task though

Reply:
I think this would be a great use case when we add an example application that depends on the various glove_box packages.